misc/python/materialize/scalability/df/paths.py

```diff
@@ -37,3 +37,11 @@ def regressions_csv_name() -> str:
 
 def regressions_csv() -> Path:
     return RESULTS_DIR / regressions_csv_name()
+
+
+def results_csv_rel_path(endpoint_name: str) -> str:
+    return f"{endpoint_name}/results.csv"
```

Comment:
There is some slight danger in joining paths like this instead of using `Path()` and the `/` operator.

Reply:
✅ Adjusted.